src/components/settings/SettingsAccessTab/index.jsx

```diff
@@ -1,50 +1,99 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { Col, Row } from '@edx/paragon';
-import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 
 import { features } from '../../../config';
 import ContactCustomerSupportButton from '../../ContactCustomerSupportButton';
 import LinkManagement from './SettingsAccessLinkManagement';
 import SSOManagement from './SettingsAccessSSOManagement';
+import SubsidyRequestManagement from './SettingsAccessSubsidyRequestManagement';
+import { SubsidyRequestConfigurationContext } from '../../subsidy-request-configuration';
 
-const SettingsAccessTab = ({ learnerPortalEnabled }) => (
-  <div className="settings-access-tab mt-4">
-    <Row>
-      <Col>
-        <h2>Enable browsing on-demand</h2>
-      </Col>
-    </Row>
-    <Row className="mb-4">
-      <Col>
-        <p>
-          Allow learners without a subsidy to browse the catalog and request enrollment to courses.
-          Browsing on demand will expire at the end of your latest subscription.
-        </p>
-      </Col>
-      <Col md="auto">
-        <ContactCustomerSupportButton variant="outline-primary">
-          Contact support
-        </ContactCustomerSupportButton>
-      </Col>
-    </Row>
-    {features.SETTINGS_UNIVERSAL_LINK && learnerPortalEnabled && (
-      <div className="mb-4">
-        <LinkManagement />
-      </div>
-    )}
-    <div className="mb-4">
-      <SSOManagement />
-    </div>
-  </div>
-);
+const SettingsAccessTab = ({
+  enterpriseId,
+  enableBrowseAndRequest,
+  enableIntegratedCustomerLearnerPortalSearch,
+  enableLearnerPortal,
+  enableUniversalLink,
+  identityProvider,
+  updatePortalConfiguration,
+}) => {
+  const {
+    subsidyRequestConfiguration,
+    updateSubsidyRequestConfiguration,
+  } = useContext(SubsidyRequestConfigurationContext);
+
+  return (
+    <div className="settings-access-tab mt-4">
+      <Row>
+        <Col>
+          <h2>Enable browsing on-demand</h2>
+        </Col>
+      </Row>
+      <Row className="mb-4">
+        <Col>
+          <p>
+            Allow learners without a subsidy to browse the catalog and request enrollment to courses.
+            Browsing on demand will expire at the end of your latest subscription.
+          </p>
+        </Col>
+        <Col md="auto">
+          <ContactCustomerSupportButton variant="outline-primary">
+            Contact support
+          </ContactCustomerSupportButton>
+        </Col>
+      </Row>
+
+      <h3>Select access channel</h3>
+      <p>
+        Channels determine how learners access the catalog(s).
+        You can select one or both and change your selection at any time.
+      </p>
 
-const mapStateToProps = state => ({
-  learnerPortalEnabled: state.portalConfiguration.enableLearnerPortal,
-});
+      {features.SETTINGS_UNIVERSAL_LINK && enableLearnerPortal && (
+        <div className="mb-4">
+          <LinkManagement />
+        </div>
+      )}
+
+      <SSOManagement
```

Comment:
nit: Maybe keep this component name the same as its file name, i.e. `SettingsAccessSSOManagement`? At first glance, I thought this might be something related to configuring SSO, not so much toggling the discovery experience in the learner portal for SSO customers.